api/v1alpha1/bucket_types.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+type Bucket struct {
```

Comment:
I don't like the name `DataProtectionBucket`, but wondering if this `Kind` gets confusing with other CRDs from ODF like 
```
bucketclasses.noobaa.io                                          
objectbucketclaims.objectbucket.io                          
objectbuckets.objectbucket.io                         
```

I think its fine... just wanted to bring it up.

Reply:
What about CloudBucket or something like  BackupStorageBucket?

Reply:
Yeah both are good with me. I also don't hate Bucket :D